core/storage/exploration/gae_models.py

```diff
@@ -239,6 +239,61 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
+class ExplorationMathRichTextInfoModel(base_models.BaseModel):
+    """Temporary Storage model for storing information useful while generating
+    images for math rich-text components in explorations.
+
+    The id of each instance is the id of the corresponding exploration.
+    """
+
+    # A boolean which indicates whether the exploration requires images to be
```

Comment:
Have you considered just deleting the model once everything has been confirmed to be generated?

Reply:
Yes I've added one-off jobs for deleting the temp models